recipes/padelpy/meta.yaml

```
@@ -0,0 +1,43 @@
+{% set name = "padelpy" %}
+{% set version = "0.1.11" %}
+
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/padelpy-{{ version }}.tar.gz
+  sha256: b490dc31c59f31d05628bd95b10d27666d689888f22043d75528914e90c7b479
+
+build:
+  number: 0
+  noarch: python
+  script: {{ PYTHON }} -m pip install . -vv
+
+requirements:
+  host:
+    - python >=3.6
+    - pip
+  run:
+    - python >=3.6
+    - openjdk
+
+test:
+  imports:
+    - padelpy
+  commands:
+    - pip check
+  requires:
+    - pip
+
+about:
+  home: https://github.com/ecrl/padelpy
+  summary: A Python wrapper for PaDEL-Descriptor
+  license: AGPL-3.0-only
+  license_file: padelpy/PaDEL-Descriptor/license/LICENSE.txt
```

It appears that license is one of several that apply to the javascript, but then the Python code is MIT. So what you need to do is add the [Python code's license]() to this PR in the recipe directory. And assuming you call it `LICENSE`, list all the licenses under `license_file`:
```suggestion
  license: AGPL-3.0-only AND LGPL-2.1-only AND MIT
  license_file:
    - padelpy/PaDEL-Descriptor/license/LICENSE.txt
    - padelpy/PaDEL-Descriptor/license/lgpl-2.1.txt
    - padelpy/PaDEL-Descriptor/license/lgpl.license
    - LICENSE
```

---

Thanks @dopplershift. 

The LICENCE file is currently not packaged in the source code (waiting for the PR in the repo to fix it).

---

@maclandrol Right, you can work around that fact by committing the file to the `recipes/padelpy` directory in this PR.